molecule/default/Dockerfile.j2

```diff
@@ -7,7 +7,8 @@ FROM {{ item.image }}
 {% endif %}
 
 RUN if [ $(command -v apt-get) ]; then apt-get update && apt-get install -y python sudo bash ca-certificates && apt-get clean; \
-    elif [ $(command -v dnf) ]; then dnf makecache && dnf --assumeyes install python sudo python-devel python2-dnf bash && dnf clean all; \
+    elif [ $(command -v dnf) ] && [ $(dnf search python2-dnf) ]; then dnf makecache && dnf --assumeyes install python sudo python-devel python2-dnf bash && dnf clean all; \
+    elif [ $(command -v dnf) ] ; then dnf makecache && dnf --assumeyes install python3 sudo python3-devel python3-dnf bash && dnf clean all; \
```

Comment:
It might make sense to synchronize the python3 case with the upstream file: https://github.com/ansible/molecule/blob/master/molecule/data/Dockerfile.j2

Reply:
Thanks, @tyll.  Indeed, we should have the upstream file.  In my local testing, I replaced  `logging/molecule/default/Dockerfile.j2` with the upstream file, then there was no problem to run `molecule converge` against centos7 and 8.  Now, does that mean since python2 EOL has passed, we can only support python3, i.e., run the tests with python3 from now on?

Having said that, there's one thing left that confuses me...  I put this debug print in the LSR/Logging tasks/main.yaml.
```
+    - name: Check python version
+      debug:
+        msg: "Python version is {{ ansible_python_version }}"
```
Then it printed the version was 2.7.5 for centos7, while 3.6.8 for centos8.  I thought molecule/default/Dockerfile.j2 installed python3 on the centos7 target host, but it was not???